TestHelper.py

```diff
@@ -61,8 +61,7 @@ def do_request(method, url, data=None):
         else:
             req = requests.Request(method, url)
         prepped = s.prepare_request(req)
-        settings = s.merge_environment_settings(prepped.url, {"http": None, "https": None},
-                                                None, CERT_TRUST_ROOT_CA, None)
+        settings = s.merge_environment_settings(prepped.url, {}, None, CERT_TRUST_ROOT_CA, None)

```

Comment:
Makes sense. Consistency good.

Since e.g. 'api.testsuite.nmos.tv' is unknown to my proxy, I probably have to remove proxy environment settings for this to work. Is that worth a ReadMe note?

Reply:
Yes, I was about to query the same thing. I'll add a Readme note, and if it becomes a pain we could potentially add a config parameter just for enabling/disabling use of the system proxy, as opposed to allowing configuration of entirely different proxy configuration.